backend/src/bundles/user-details/types/user-details-properties.type.ts

```diff
@@ -14,6 +14,7 @@ type UserDetailsProperties = {
     id: string | null;
     userId: string;
     isApproved: boolean;
+    isPublished: boolean;
```

Comment:
we need to have date in publish field, to show it on employer page: 
<img width="512" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-bsa-talents/assets/85788646/43dbc4ff-6f7b-4ee5-96c4-7ee1c0755ce1">